### PR TITLE
chore(lidar_centerpoint): updated centerpoint models and documentation

### PR DESCRIPTION
**Pull request by knzo25**
Signed-off-by: Kenzo Lobos-Tsunekawa <kenzo.lobos@tier4.jp>

## Description
Add new centerpoint models trained in argoverse2 and nuscenes

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [ ] I've confirmed the [contribution guidelines].
- [ ] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: perception/lidar_centerpoint/README.md [hunk 1]
```diff
@@ -52,6 +52,12 @@ We trained the models using <https://github.com/open-mmlab/mmdetection3d>.
 
 You can download the onnx format of trained models by clicking on the links below.
 
+- [Centerpoint](download_link)
+- [Centerpoint tiny](download_link)
```
Comment (yukke42) [inline perception/lidar_centerpoint/README.md:56]: Cloud you update the link?

File: perception/lidar_centerpoint/README.md [hunk 2]
```diff
@@ -52,6 +52,12 @@ We trained the models using <https://github.com/open-mmlab/mmdetection3d>.
...
+
+`Centerpoint` was trained in `Nuscenes` (~110k lidar frames) [8] and Tier IV's internal database (~11k lidar frames) for 60 epochs.
+`Centerpoint tiny` was trained in `Argoverse2` (~28k lidar frames) [9] and Tier IV's internal database (~11k lidar frames) for 20 epochs.
```
Comment (yukke42) [inline perception/lidar_centerpoint/README.md:59]:
```suggestion
`Centerpoint` was trained in `nuScenes` (~110k lidar frames) [8] and TIER IV's internal database (~11k lidar frames) for 60 epochs.
`Centerpoint tiny` was trained in `Argoverse 2` (~28k lidar frames) [9] and TIER IV's internal database (~11k lidar frames) for 20 epochs.
```

File: perception/lidar_centerpoint/README.md (hunk 1 above)
Comment (knzo25) [reply perception/lidar_centerpoint/README.md]: done !

**Review by yukke42 — APPROVED** [review]
LGTM

**Pull request merged**